### ISSUE TITLE
Add support for multiple languages

### ISSUE THREAD

**Issue by lmachens** [lmachens/aeternum-map#62]
Translations are easy to implement for static texts, but I am not sure how to translate comments or custom texts like character names.
Any ideas?

**Comment by ivan-cavero** [comment]
Yes, they have different names, but I suppose that this could be changed little by little, the interface and so on would be easier.
I think the names of NPCs are the same, because they are personal names.
**For example:**
In spanish = PUERCO DE AURORA
![image](https://user-images.githubusercontent.com/32484621/137628669-e54210cc-d809-4074-9cc3-375df383e450.png)
In English = DAYSPRING SWINE
![image](https://user-images.githubusercontent.com/32484621/137628684-8e2d29c2-0973-4265-9a16-b224f9a302cf.png)
